packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/java/callbackuriwithotpandstate.md

```diff
@@ -0,0 +1 @@
+`http://localhost:8080/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}`
```

Comment:
Same question as above.

Reply:
FWIW this _is_ a valid link in Arvind's sample app